packages/kbn-search-api-panels/types.ts

```diff
@@ -37,6 +37,7 @@ export interface LanguageDefinition {
   configureClient?: CodeSnippet;
   ingestData?: CodeSnippet;
   ingestDataIndex?: CodeSnippet;
+  ingestDataWithPipeline?: CodeSnippet;
```

Comment:
Do we need this? in the code snippets we just added pipeline as an optional argument for `ingestData` & `ingestDataIndex`. Can we just do the same for the console snippet and not add a 3rd ingestData option?

Reply:
Updated.